blocks/resources/block.php

```diff
@@ -50,6 +51,17 @@ function render_callback( array $attributes ) : string {
 	$resources_query = new AirpressQuery( 'Resources', 0 );
 	$resources_query->addFilter( '{Publish Status of Resource} = "Published"' );
 
+	$trp           = TRP_Translate_Press::get_trp_instance();
+	$url_converter = $trp->get_component( 'url_converter' );
+	$language_code = $url_converter->get_lang_from_url_string();
+
+	if ( $language_code ) {
+		$language_handler = $trp->get_component( 'languages' );
+		$language_name    = $language_handler->get_language_names( array( $language_code ), 'english_name' )[ $language_code ];
+		$resources_query->view( 'Published (' . $language_name . ') Resources grouped by Resource Type - site embed view' );
```

Comment:
Do we always know that the language will be in this table?

Reply:
We should add future languages to airtable at the same time as we add them to the translatepress language switcher. We can set each airtable view so that it shows something reasonable (eg the english entries) even if the translators haven't finished translating yet.

Reply:
Got it. I'm just worried as this code will fail if the language does not exist, and I think we should be a touch more defensive here.